### DISCUSSION TITLE
📢 Welcome to discuss everything related with 🐥 Cyberduck & 🦆 Mountain Duck for macOS and Windows

### DISCUSSION THREAD

**Discussion by dkocher** [iterate-ch/cyberduck#14309] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other users of [🐥 Cyberduck](https://cyberduck.io/) and [🦆 Mountain Duck](https://mountainduck.io/). We hope that you:
  * Ask questions you’re wondering about.
  * Help others with problems related to their setup
  * Share use cases and ideas
  * Engage with other community members.
  
Welcome others and are open-minded. Remember that this is a community we build together 💪. This replaces the previous [Google Group](https://groups.google.com/g/cyberduck) now archived.

**Comment by RobertCailliau** [comment]


**Reply by dkocher** [reply]
Please create a new discussion instead of adding a comment here.

**Comment by THara707** [comment]
Dear Sir,

I recently purchased Cyberduck with a donation.
I received the license key, clicked on it to activate the software, and was able to access the server.

However, when I drag files from my PC's desktop to upload them, they don't transfer into the server.

What should I do?

Tatsuya Hara
